content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,361 @@
-==================================================
-What is a Removal Strategy (FIFO, LIFO, and FEFO)?
-==================================================
-
-Usually, *Removal Strategies* are defined in picking operations to select the best products to
-optimize the distance for the worker, for quality control purposes, or to first move the products
-with the closest expiration date.
-
-When a product movement needs to be done, Odoo finds available products that can be assigned to
-the transfer. The way Odoo assigns these products depends on the *Removal Strategy* defined in
-the *Product Category* or on the *Location*.
+=====================================
+Removal Strategies (FIFO, LIFO, FEFO)
+=====================================
+
+For companies with warehouses, **removal strategies** determine which products are taken from the
+warehouse, and when. Removal strategies are typically defined for specific picking operations. This
+helps companies to select the best products, optimize the distance workers need to travel when
+picking items for orders, and account for quality control, such as moving products with expiration
+dates.
+
+When a product needs to be moved, Odoo finds available products that can be assigned to the
+transfer. The way Odoo assigns these products depends on the :guilabel:`Removal Strategy` defined in
+either the :guilabel:`Product Category` or the :guilabel:`Location`.
+
+To change the :guilabel:`Removal Strategy`, go to :menuselection:`Inventory app --> Configuration
+--> Locations` or :guilabel:`Product Categories`. Click on a :guilabel:`Location` or
+:guilabel:`Product Category`, and then click :guilabel:`Edit`. Change the product category
+:guilabel:`Force Removal Strategy` or the location :guilabel:`Removal Strategy` by clicking on the
+drop-down menu and selecting the desired removal strategy. After selecting the new removal strategy,
+click :guilabel:`Save`.
+
+.. image:: removal/product-category-location.png
+   :align: center
+   :alt: Change the Force Removal Strategy for either the Product Categories or Locations.
 
 What happens inside the warehouse?
 ==================================
 
-Imagine a generic warehouse plan, with receiving docks and area, storage locations, picking and
-packing areas, and shipping docks. All products go through all these locations, but some rules,
-such as removal strategies, can have an effect on which products are taken for the pickings.
+Most warehouses share the same important areas: receiving docks and sorting areas, storage
+locations, picking and packing areas, and shipping/loading docks. While all products entering or
+leaving the warehouse might go through each of these locations at some point, removal strategies can
+have an effect on which products are taken, from where, and when.
 
 .. image:: removal/empty-dock.png
```

Comment:
Image needs to be resized

Reply:
I have NO idea how these got messed up! I retook them and I *know* I made them 933.... yikes. I'm sorry- I don't understand what happened (unless I saved them on my local instead of the github image folder!) Apologies- I know better than this =)